run-tests.sh

```diff
@@ -1,14 +1,63 @@
-#!/bin/sh
+#!/bin/bash
 #
 # This file is part of REANA.
 # Copyright (C) 2018 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
+export REANA_SQLALCHEMY_DATABASE_URI=postgresql+psycopg2://postgres:mysecretpassword@localhost/postgres
+
+# Verify that db container is running before continuing
+_check_ready() {
+    RETRIES=40
+    while ! $2
+    do
+        echo "==> [INFO] Waiting for $1, $((RETRIES--)) remaining attempts..."
+        sleep 2
+        if [ $RETRIES -eq 0 ]
+        then
+            echo "==> [ERROR] Couldn't reach $1"
+            exit 1
+        fi
+    done
+}
+
+_db_check() {
+    docker exec --user postgres reana-postgres bash -c "pg_isready" &>/dev/null;
+}
+
+clean_old_db_container() {
+    OLD="$(docker ps --all --quiet --filter=name=reana-postgres)"
+    if [ -n "$OLD" ]; then
+        echo '==> [INFO] Cleaning old DB container...' && \
```

Comment:
We don't need `&&` in these internal statements after `echo`.  It is sufficient if `&&` were enchained only at the bottom of the script when calling them.

Actually, it would be simpler to delete `&&` everywhere, since we can do "stop-after-first-failure" by setting:
```
# Quit on errors
set -o errexit
```

in the beginning of the script.

P.S. We could also add the following for shell script security:

```
# Quit on unbound symbols
set -o nounset
```

at the top as well.